external/source/exploits/CVE-2021-40449/CVE-2021-40449/dllmain.c

```diff
@@ -0,0 +1,684 @@
+#pragma warning( disable : 4005 )
+
+#define REFLECTIVEDLLINJECTION_VIA_LOADREMOTELIBRARYR
+#define REFLECTIVEDLLINJECTION_CUSTOM_DLLMAIN
+#include "ReflectiveLoader.c"
+#include <stdio.h>
+#include <winddi.h>
+#include <Windows.h>
+#include <psapi.h>
+#include <tlhelp32.h>
+#include <winspool.h>
+#include "../../include/windows/common.h"
+
+#define SystemHandleInformation         0x10
+#define SystemBigPoolInformation        0x42

```

Comment:
These two should also be defined in the `SYSTEM_INFORMAITON_CLASS` enum, and are already in the `definitions.h` file I mentioned in the other comment.

Reply:
Yep, sounds good.